WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -707,6 +726,10 @@ import AutomatticTracks
 
     /// Scrolls to the top of the list of posts.
     @objc func scrollViewToTop() {
+        guard !FeatureFlag.readerReset.enabled else {
```

Comment:
This fixes an issue where after selecting interests from a prompt, it would scroll in a weird spot. I don't think it needs to scroll at all, so, for simplicity, I'm removing it.